itest/overcast-docker-itest/overcast.conf

```diff
@@ -4,10 +4,30 @@
 //   dockerHost=http://localhost:2375
 // }
 
-dockerMinimalConfig {
+dockerMinimalConfig {   
   dockerImage="busybox"
   command=["/bin/sh", "-c", "while true; do echo hello world; sleep 1; done"]
   remove=true
+  removeVolume=true
```

Comment:
I've added the removeVolume flag to avoid having to manually invoke _docker volume rm_ - I noticed that orphaned volumes keep piling up after each run causing my VM's HDD to keep increasing in size (albeit insignificantly).